cmd/bucket-lifecycle.go

```diff
@@ -76,6 +76,8 @@ type expiryState struct {
 
 func (es *expiryState) queueExpiryTask(oi ObjectInfo, rmVersion bool) {
 	select {
+	case <-GlobalContext.Done():
+		close(es.expiryCh)
```

Comment:
suggest just returning on GlobalContext.Done() and leaving the go routine that closes the expiryCh in newExpiryState as is. Otherwise, this will result in panic of closing an already closed channel

Reply:
Yes return is missing here.